spec/requests/api/v1/course_spec.rb

```diff
@@ -49,12 +49,13 @@ def get_course_codes_from_body(body)
           provider: provider,
           age: 2.hours.ago)
 
-        course.site_statuses.first.update(
+        course.site_statuses.first.update_attributes(
```

Comment:
These changes were needed because the `changed_at` gets updated implicitly when creating the course with the factory.

Reply:
`#update_attributes` is an alias for `#update`, and looks like it's actually been deprecated in favour of the later:

https://github.com/rails/rails/blob/35efe44f5379b031e6d4abab7b0a8c280e31b640/activerecord/lib/active_record/persistence.rb#L641-L642

```ruby
    alias update_attributes update
    deprecate update_attributes: "please, use update instead"
```

Is there a reason for changing this to `update_attributes`?

Reply:
Ah yes. We should configure our linter to tell us about that. It's a default in Rubocop, not sure why it is disabled.